packages/components/src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -13,17 +14,21 @@ interface DropdownMenuProps extends React.HTMLProps<HTMLDivElement> {
 interface DropdownMenuItemProps extends React.HTMLProps<HTMLDivElement> {
   children?: React.ReactNode;
   className?: string;
+  isLoading?: boolean;
```

Comment:
hey @AnthonyDS-symphony do you mind aligning it with button loading state and rename it as "loading"